apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx

```diff
@@ -379,11 +379,7 @@ export function PollWorkerScreen({
                           key={precinct.id}
                           aria-label={`Activate Voter Session for Precinct ${precinct.name}`}
                           onPress={() => {
-                            setAcceptingPaperStateMutation.mutate(undefined, {
-                              onSuccess() {
-                                setSelectedCardlessVoterPrecinctId(precinct.id);
-                              },
-                            });
```

Comment:
This was redundant with another call to `setAcceptingPaperStateMutation.mutate` that happens after ballot style selection